Docs/user_guide/getting_started.md

```diff
@@ -149,3 +149,51 @@ Browse the [User Guide](user_interface) section to find quick overview of the ap
 
 **If you have any questions, go to the [Slicer forum](https://discourse.slicer.org) and ask us!**
+
+## Glossary
+
+Terms used in various fields of medical and biomedical image computing and clinical images are not always consistent. This section defines terms that are commonly used in 3D Slicer, especially those that may have different meaning in other contexts.
+
+- Annotation: Simple geometric objects and measurements that the user can place in viewers. Annotations module can be used to create such objects, but it is deprecated - being replaced by markups.
+- Bounds: Describes bounding box of a spatial object along 3 axes. Typically defined by 6 floating-point values: X_min, X_max, Y_min, Y_max, Z_min, Z_max.
+- Brightness/contrast: Specifies linear mapping of voxel values to brightness of a displayed pixel. Brightness is the linear offset, contrast is the multiplier.
+- Cell: Data cells are simple topological elements like lines, polygons, tetrahedra, etc.
+- Coordinate system (or coordinate frame, reference frame, space): Specified by position of origin, axis directions, and distance unit. All coordinate systems in 3D Slicer are right-handed.
+- Extension: A collection of modules that is not bundled with the core application but can be downloaded and installed using the Extensions manager.
+- [Extensions manager](extensions_manager): A software component of Slicer that allows browsing, installing, uninstalling extensions in the [Extensions catalog (also known as the Slicer app store)](http://slicer.kitware.com/midas3/slicerappstore) directly from the application.
+- [Extensions index](https://github.com/Slicer/ExtensionsIndex): A repository that contains description of each extension that the Extension catalog is built from.
+- Extent: Range of integer coordinates along 3 axes. Typically defined by 6 values, for IJK axes: I_min, I_max, J_min, J_max, K_min, K_max. Both minimum and maximum values are inclusive, therefore size of an array is (I_max - I_min + 1) x (J_max - J_min + 1) x (K_max - K_min + 1).
+- Fiducial: Represents a point in 3D space. The terms originates from image-guided surgery, where "fiducial markers" are used to mark point positions.
+- Frame: One time point in a time sequence. To avoid ambiguity, this term is not used to refer to a slice of a volume.
+- Geometry: Specify location and shape of an object in 3D space.
+- Intensity: Typically refers to value of a voxel.
+- IJK: Voxel coordinate system axes. Each integer coordinate value combination corresponds to a voxel center.
+- ITK: [Insight Toolkit](https://itk.org/). Software library that Slicer uses for most image processing operations.
+- Labelmap (or labelmap volume, labelmap volume node): Volume node that has discrete (integer) voxel values. Typically each value corresponds to a specific structure or region. This allows compact representation of non-overlapping regions in a single 3D array.
```

Comment:
Could also indicate the relationship with segment managed in the segment editor ?